### PR TITLE
added datetimepicker in create task

### PR DESCRIPTION
**Pull request by aginzberg**
Integrates datetime picker from https://github.com/Eonasdan/bootstrap-datetimepicker. If we end up using this, we should send him an email letting him know.

### CODE REVIEW

**Comment by neilthemathguy** [comment]
For the date-time picker we can use: https://angular-ui.github.io/bootstrap/

**Comment by dmorina** [comment]
Error: [ngModel:nonassign] Expression 'project.startDate | date:'MM/dd/yyyy h:mm'' is non-assignable. Element: <input class="input-group date ng-pristine ng-untouched ng-valid" id="datetimepicker1" required="" ng-model="project.startDate | date:'MM/dd/yyyy h:mm'">

File: staticfiles/templates/project/project.html [hunk 1]
```diff
@@ -46,18 +47,28 @@ <h1 class="md-header-darkgray">Create a New Project</h1>
...
-                        <div ng-if="project.startDate==null" ng-messages="projectForm.description.$error">
-                            <div ng-message="required">This is required.</div>
-                        </div>
+                      <label>Start Date</label>
+                      <input class='input-group date' id='datetimepicker1' required ng-model="project.startDate | date:'MM/dd/yyyy h:mm'">
+                      <div ng-if="project.startDate==null" ng-messages="projectForm.description.$error">
+                        <div ng-message="required">This is required.</div>
+                      </div>
+                      <script type="text/javascript">
+                          $(function () {
```
Comment (nistala) [inline staticfiles/templates/project/project.html]: don't use jquery

File: staticfiles/templates/project/project.html [hunk 2]
```diff
@@ -46,18 +47,28 @@ <h1 class="md-header-darkgray">Create a New Project</h1>
...
-                      <input type="datetime" required ng-model="project.endDate">
-                        <div ng-if="project.endDate==null" ng-messages="projectForm.description.$error">
-                            <div ng-message="required">This is required.</div>
-                        </div>
+                      <input class='input-group date' id='datetimepicker2' required ng-model="project.endDate | date:'MM/dd/yyyy h:mm'">
+                      <div ng-if="project.endDate==null" ng-messages="projectForm.description.$error">
+                        <div ng-message="required">This is required.</div>
+                      </div>
+                      <script type="text/javascript">
+                          $(function () {
```
Comment (nistala) [inline staticfiles/templates/project/project.html]: you shouldn't insert script tags into the code, please use http://mgcrea.github.io/angular-strap/#/datepickers, this should offer the required functionality in an angular centric way

**Pull request closed without merging**